src/components/spreadsheet/spreadsheet.ts

```diff
@@ -222,6 +225,23 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
       startCellEdition: (content: string) => this.onGridComposerCellFocused(content),
     });
 
+    useEffect(
+      () => {
+        /**
+         * Only refocus the grid if the active element is not a child of the spreadsheet
+         * and spreadsheet is a child of that element. Anything else means that the focus
+         * is on an element that needs to keep it.
+         */
+        if (
+          !this.spreadsheetRef.el!.contains(document.activeElement) &&
+          document.activeElement?.contains(this.spreadsheetRef.el!)
```

Comment:
for future reference, this means "activeElement is outside the spreadsheetRef component"